delivery/app.py

```diff
@@ -47,6 +49,8 @@ def routes(**kwargs):
         url(r"/api/1.0/projects", ProjectHandler, name="projects", kwargs=kwargs),
         url(r"/api/1.0/runfolders/(.+)/projects", ProjectsForRunfolderHandler,
             name="projects_for_runfolder", kwargs=kwargs),
+        url(r"/api/1.0/project/(.+)/best_practice_samples", BestPracticeProjectSampleHandler,
```

Comment:
is it necessary to explicitly exclude slashes in the regexp to make sure that the endpoint is unambiguous? i.e. could the regexp above match both `.../project/A/best_practice_samples` and `.../project/A/B/best_practice_samples`?